Sofa/framework/Testing/CMakeLists.txt

```diff
@@ -4,10 +4,54 @@ project(Sofa.Testing LANGUAGES CXX)
 # Enable testing features of cmake, like the add_test() command.
 enable_testing()
 
-# add gtest library
-# set(gtest_disable_pthreads ON CACHE INTERNAL "")
-add_subdirectory(extlibs/gtest)
-# add custom gtest main
+# add googletest library
+# set(googletest_disable_pthreads ON CACHE INTERNAL "")
+find_package(googletest QUIET)
```

Comment:
CMake config/module files uses the GTest namespace for googletest:
https://cmake.org/cmake/help/latest/module/FindGTest.html

Reply:
`find_package(GTest QUIET)`